monasca/README.md

```diff
@@ -228,6 +224,41 @@ Parameter | Description | Default
 `keystone.resources.limits.memory` | Memory limit per keystone pod | `1Gi`
 `keystone.resources.limits.cpu` | Memory limit per keystone pod | `500m`
 
+
+### MySQL
+
+Parameter | Description | Default
+----------|-------------|--------
+`mysql.imageTag` | Tag to use from `library/mysql` | `5.6`
+`mysql.imagePullPolicy` | K8s pull policy for mysql image | `IfNotPresent`
+`mysql.persistence.enabled` | If `true`, enable persistent storage | `false`
+`mysql.persistence.storageClass` | K8s storage class to use for persistence | `default`
+`mysql.persistence.accessMode` | PVC access mode | `ReadWriteOnce`
+`mysql.persistence.size` | PVC request size | `10Gi`
+`mysql.resources.requests.memory` | Memory request | `256Mi`
+`mysql.resources.requests.cpu` | CPU request | `100m`
+`mysql.resources.limits.memory` | Memory limit | `1Gi`
+`mysql.resources.limits.cpu` | CPU limit | `500m`
+`mysql.users.keystone.username` | Keystone MySQL username | `keystone`
+`mysql.users.keystone.password` | Keystone MySQL password | `keystone`
+`mysql.users.api.username` | API MySQL username | `monapi`
+`mysql.users.api.password` | API MySQL password | `password`
+`mysql.users.notification.username` | Notification MySQL username | `notification`
+`mysql.users.notification.password` | Notification MySQL password | `password`
+`mysql.users.thresh.username` | Thresh MySQL username | `thresh`
+`mysql.users.thresh.password` | Thresh MySQL password | `password`
+
+
+### MySQL Init
```

Comment:
They are ... at least at the time it seemed better to keep them under `mysql` since they don't just apply at init time and are now used across all services that need mysql credentials. (also it's shorter!)

It would be relatively easy to change if you'd prefer.